src/functions/template-tags/general.php

```diff
@@ -1450,7 +1450,7 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 
 		if ( ! has_excerpt( $post->ID ) ) {
 			// Temporarily alter the global post in preparation for our filters.
-			$global_post     = $GLOBALS['post'];
+			$global_post = isset( $GLOBALS['post'] ) ? $GLOBALS['post'] : '';
```

Comment:
I'll defer to your judgement here, but if `$GLOBALS['post']` isn't set, might 'restoring it' to null be safer and more reflective of what it was (than setting it to an empty string)?

Mostly, I'm sure it won't matter, but if there _is_ code out there that might run immediately after this and does it's own `isset( $GLOBALS['post'] )` then we'll be triggering a different path of execution in that code.